index.js

```diff
@@ -1,7 +1,10 @@
-var window              = require('global');
-var MockXMLHttpRequest  = require('./lib/MockXMLHttpRequest');
-var real                = window.XMLHttpRequest;
-var mock                = MockXMLHttpRequest;
+var Promise                 = require('lie');
+var window                  = require('global');
+var MockXMLHttpRequest      = require('./lib/MockXMLHttpRequest');
+var real                    = window.XMLHttpRequest;
+var mock                    = MockXMLHttpRequest;
+
+window.xhrMockProxyHandler  = require('./lib/xhr-mock-proxy-handler');
```

Comment:
If the user is using browserify/webpack they'll just be able to `require`/`import` it.
```
const xhrMock = require('xhr-mock');
const xhrMockProxyHandler = require('xhr-mock-proxy-handler');
```
If the user is using the library directly in the browser without browserify/webpack we'll need to use browserify/webpack/rollup to generate a UMD.
```
<script src="path/to/xhr-mock/dist/xhr-mock.min.js"></script>
<script src="path/to/xhr-mock-proxy-handler/dist/xhr-mock-proxy-handler.min.js"></script>
<script>
window.XHRMock;
window.XHRMockProxyHandler
</script>
```

Feel free to keep developing the handler as part of this PR so I can review it, then you can move it to its own package right before I merge the PR.